debug-scripts/etcd

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -ux
+
+systemctl status etcd > $DEBUG_SCRIPT_DIR/etcd-systemctl-status
+journalctl -u etcd > $DEBUG_SCRIPT_DIR/etcd-journal
+
+alias etcdctl="etcdctl --cert-file /etc/ssl/etcd/client.crt --key-file /etc/ssl/etcd/client.key --ca-file /etc/ssl/etcd/ca.crt"
+etcdctl cluster-health > $DEBUG_SCRIPT_DIR/etcdctl-cluster-health
+etcdctl ls --recursive > $DEBUG_SCRIPT_DIR/etcdctl-ls
+etcdctl member list > $DEBUG_SCRIPT_DIR/etcdctl-member-list
```

Comment:
Certainly. I think it makes sense to live there so long as we are parsing out the locations from the layer.yaml.

It became a little more obvious to me the more I thought about this that we're going to have to rely on the implementor to actually supply the TLS locations for x509 validation. Thats potentially problematic as I don't know that we can reasonably infer what layer it was included in.

I think it needs more thought and perhaps we can get a dict back of the layer options and just iterate over them to discover which ones have tls-cert keys and can then run the gambit of validations there.

Thanks @Cynerva for the submission. I'm good with this as is and will gladly take further patches to enhance functionality.